site/layouts/shortcodes/helm-version.html

```diff
@@ -6,8 +6,8 @@
 {{- "v1.1.4" -}}
 {{- end -}}
 {{- with (strings.HasPrefix $pagePrefix "v1.2") -}}
-{{- "v1.2.2" -}}
+{{- "v1.2.3" -}}
```

Comment:
Should this be part of another PR, else this will cause the.docs.site to be broken for some duration

Reply:
A better approach could be releasing first, then modify the site on the main branch.

Reply:
I think a better approach would be publishing the release first, then modify the doc?

Reply:
Yeah +1